cmake/usage/CMakeLists.txt

```diff
@@ -3,6 +3,18 @@
 string(REPLACE ";" " " BOARD_ROOT_SPACE_SEPARATED "${BOARD_ROOT}")
 string(REPLACE ";" " " SHIELD_LIST_SPACE_SEPARATED "${SHIELD_LIST}")
 
+string(REPLACE " " ";" SHIELD_LIST "${SHIELD_LIST_SPACE_SEPARATED}")
+set(sorted_shield_list ${SHIELD_LIST})
+list(SORT sorted_shield_list)
+set(sorted_shield_cmds)
+foreach(shield ${sorted_shield_list})
+  list(APPEND sorted_shield_cmds COMMAND ${CMAKE_COMMAND} -E echo "  ${shield}")
+endforeach()
+add_custom_target(shields ${sorted_shield_cmds})
+
+include (${ZEPHYR_BASE}/cmake/boards.cmake)
+add_custom_target(boards ${list_boards_commands} USES_TERMINAL)
```

Comment:
I think creation of this target should happen directly inside: `cmake/boards.cmake`

Reply:
> I think creation of this target should happen directly inside: `cmake/boards.cmake`

add_custom_target() is not available in script mode, and boards.cmake still supports that.

Reply:
good point. :+1: